pkg/rules/rego/deprecated-1-22.rego

```diff
@@ -41,11 +46,36 @@ deprecated_api(kind, api_version) = api {
 			"new": "coordination.k8s.io/v1",
 			"since": "1.19",
 		},
-		"LeaseList": {
-			"old": ["coordination.k8s.io/v1beta1"],
-			"new": "coordination.k8s.io/v1",
+		"PriorityClass": {
+			"old": ["scheduling.k8s.io/v1beta1"],
+			"new": "scheduling.k8s.io/v1",
+			"since": "1.14",
+		},
+		"RBAC": {
+			"old": ["rbac.authorization.k8s.io/v1beta1"],
+			"new": "rbac.authorization.k8s.io/v1",
+			"since": "1.8",
+		},
+		"CertificateSigningRequest": {
+			"old": ["certificates.k8s.io/v1beta1"],
+			"new": "certificates.k8s.io/v1",
 			"since": "1.19",
 		},
+		"APIService": {
+			"old": ["apiregistration.k8s.io/v1beta1"],
+			"new": "apiregistration.k8s.io/v1",
+			"since": "1.10",
+		},
+		"CustomResourceDefinition": {
+			"old": ["apiextensions.k8s.io/v1beta1"],
+			"new": "apiextensions.k8s.io/v1",
+			"since": "1.16",
+		},
+		"Webhooks": {
```

Comment:
This actually needs to be split in 2 `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` resources.